kDriveCore/Data/UploadQueue/Operation/UploadOperation+PHAsset.swift

```diff
@@ -56,4 +57,22 @@ extension UploadOperation {
             file.pathURL = url
         }
     }
+
+    func checkForRestrictedUploadOverDataMode() throws {
```

Comment:
I'm not found of this, can we keep it in UploadQueue ? We do not want to put each upload in error, do we ?

The behaviour until now was the upload queue would not start. We need to discuss this together at least.

Reply:
Discussed, we keep some logic on the `UploadOperation` side but also on the `UploadQueue` for a global pause of the uploads.